src/app/app.component.html

```diff
@@ -8,6 +8,14 @@
       >
     </div>
     <div class="links">
+      <a target="_blank" href="/factorio">
```

Comment:
Using `target="_blank"` opens the link in a new tab. In this case I think we want to reload the current page, which is more like the title `<a>` tag, which includes the `setTimeout` call to reload the page.

Reply:
I would prefer to put these in a separate group of links next to the title, like in the Factorio website itself:
![image](https://user-images.githubusercontent.com/8430702/123846401-52027280-d8e3-11eb-81ae-ab5536238455.png)

Reply:
I might have to experiment with it, but I think the `href` might work better without the leading `/`. I think this works fine in production but in staging the link would redirect to production. Leaving out the leading `/` lets you link without sending to production, I think. If you note, that's what the `homeHref` function does in `app.component.ts`.

Reply:
done